proposed/0013-immutable-data-naming/0013-immutable-data-naming.md

```diff
@@ -13,37 +13,28 @@ This RFC outlines the system components and design for how the three immutable d
+The SAFE Network keeps multiple copies of a single ImmutableData chunk, not only for availability but also for security reasons.  To that end, three different types of ImmutableData have been defined: Normal, Backup and Sacrificial.  These differ only in how their name is calculated; their contents are identical.
 
-Making three different types copy bearing different but determinic name will increase the difficultiy of any possible attack, as it will be three different groups, instead of just one group, need to be tackled at the same time.
+Having three different types bearing different but deterministic names will increase the difficulty of any attack, as it will be three different groups of Vault, which need to be tackled at the same time.
 
 ## Supported Use-Cases
 
-For client perspective, the use-case keeps the same : storing or fetching immutable data.  Though client may need to be aware of the existing of different types and calculations of name. 
+From the client perspective, the use-case stays the same: storing or fetching immutable data.  However, the client may need to be aware of the existence of different types.
 
 ## Expected Outcome
 
-The different immutable data type copy bearing different name, which will be handled by different DataManager group.
+Replication of the different immutable data types bearing different names, which will be handled by different DataManager groups.
 
 # Detailed design
 
 ## Overview
 
-The name of an immutalbe data copy will be based on it's type and in different hash order :
+For each type (Normal, Backup and Sacrificial) of a given chunk, the management will be done by the ImmutableDataManagers closest to the chunk's name, and a copy will be stored on two PmidNodes within that close group.
 
-Normal : normal_name = hash(im.content), handled by DM(normal_name), 4 copies on the pmid_nodes picked up by that group (one copy on each node)
-
-Backup : backup_name = hash(normal_name), handled by DM(backup_name), 4 copies on the pmid_nodes picked up by that group (one copy on each node)
-
-Sacrificial : sacrificial_name = hash(backup_name), handled by DM(sacrificial_name), 4 copies on the pmid_nodes picked up by that group (one copy on each node)
-
-MaidManagers will charge the client issuing the put request with 4 copies of the data, as the backup and sacrificial copies are allowed to be removed from the SAFE network according to the network status.
-
-The portal DMs (DataManagers that being the closest group to narmal_name), fowards the put/get requests to the second/third order DMs (i.e. DataManagers closing to backup_name and sacrificial_name). This reduces the exposure of network data types to client side to minimum.
+The Normal DMs (DataManagers comprising closest group to the Normal name), forward client Put/Get requests to the Backup and Sacrificial DMs.  This minimises client exposure to the network data types and their management.
```

Comment:
Only if they cannot themselves retrieve from their own group. This is a churn related error and should be addressed there.